### PR TITLE
Add diagram for clinic

### PR DESCRIPTION
**Pull request by imranhossainemi**
Project update for clinic diagram : 

- Diagram of vet clinic
>
clinic_diagram

- Create schema_based_on_diagram.sql file to implement the database from the diagram

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
- [ ] you have requested a review for [this project](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md) as you can see this is a different diagram than the one you used. please make sure to use the right diagram for this milestone.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by imranhossainemi** [comment]
@mikethreels Thank you

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+CREATE TABLE treatments(
+id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY REFERENCES medical_histories(id),
+type VARCHAR,
+name VARCHAR,
+);
+
+ALTER TABLE medical_histories ADD CONSTRAINT fk_id FOREIGN KEY (id) REFERENCES treatments(id);
+
+ALTER TABLE treatments ADD CONSTRAINT fk_id FOREIGN KEY (id) REFERENCES medical_histories(id);
```
Comment (pinkmoon25) [inline schema_based_on_diagram.sql:31]: - [ ] [OPTIONAL] Good job so far, however I think you are missing the Join table for the many to many relation between `medical_histories` and `treatments` tabel, I am making it as optional, as it was not mentioned earlier but please do implment it as it is a core requirement :+1: 
something like   
```
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

**Review by pinkmoon25 — APPROVED** [review]
Hi @imranhossainemi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
gitflow :heavy_check_mark: 
Pair-programming :heavy_check_mark: 
Schema diagram updates :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**